server/cms/admin.py

```diff
@@ -229,7 +229,7 @@ class SurveyResponseResource(resources.ModelResource):
 
     class Meta:
         model = SurveyResponse
-        export_order = ('id', 'session_id', 'name', 'date_updated')
+        export_order = ('id', 'session_id', 'date_updated')
```

Comment:
Do we not need the zip code here? I feel like that's the only thing (other than the date) that can really be analyzed.

Reply:
@bk3c confirming that the zipcode is present (tested the CSV export)